### PR TITLE
Feature/downgrade compose top base activity

### PR DESCRIPTION
**Pull request by daniloqueirooz**
downgrade activity model

### CODE REVIEW

File: app/src/androidTest/java/io/glassgreenhouse/smartgarden/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package io.glassgreenhouse.smartgarden
+
```
Comment (ismaelqueiroz) [inline app/src/androidTest/java/io/glassgreenhouse/smartgarden/ExampleInstrumentedTest.kt]: refactory/rename java package to kotlin

**Comment by daniloqueirooz** [comment]
downgrade to base activity model

**Review by ismaelqueiroz — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/io/glassgreenhouse/smartgarden/ExampleInstrumentedTest.kt (hunk 1 above)
Comment (ismaelqueiroz) [reply app/src/androidTest/java/io/glassgreenhouse/smartgarden/ExampleInstrumentedTest.kt]: no, the package still persists as java

**Comment by daniloqueirooz** [comment]
I refactored, I will commit

**Review by ismaelqueiroz — APPROVED** [review]

**Pull request merged**